cirq-core/cirq/ops/phased_x_z_gate_test.py

```diff
@@ -199,6 +199,22 @@ def test_from_matrix_close_unitary(unitary: np.ndarray):
     )
 
 
+@pytest.mark.parametrize(
+    'unitary',
+    [
+        cirq.testing.random_unitary(2),
+        cirq.testing.random_unitary(2),
+        cirq.testing.random_unitary(2),
+        np.array([[0, 1], [1j, 0]]),
+    ],
+)
+def test_from_matrix_close_kraus(unitary: np.ndarray):
+    gate = cirq.PhasedXZGate.from_matrix(unitary)
+    kraus = cirq.kraus(gate)
+    assert len(kraus) == 1
+    cirq.testing.assert_allclose_up_to_global_phase(kraus[0], unitary, atol=1e-8)
```

Comment:
Question: is this tol too high for complex64. I would've expected only around 6-7 decimal digits of reliability.

Reply:
This was copy-pasted from other `allclose` checks in the same module. I'm inclined to keep it for consistency, but we could certainly relax it throughout the module if desired.